crates/bevy_pbr/src/render/rgb9e5.wgsl

```diff
@@ -0,0 +1,63 @@
+#define_import_path bevy_pbr::rgb9e5
+
+const RGB9E5_EXPONENT_BITS        = 5u;
+const RGB9E5_MANTISSA_BITS        = 9;
+const RGB9E5_MANTISSA_BITSU       = 9u;
+const RGB9E5_EXP_BIAS             = 15;
+const RGB9E5_MAX_VALID_BIASED_EXP = 31u;
+
+//#define MAX_RGB9E5_EXP               (RGB9E5_MAX_VALID_BIASED_EXP - RGB9E5_EXP_BIAS)
+//#define RGB9E5_MANTISSA_VALUES       (1<<RGB9E5_MANTISSA_BITS)
+//#define MAX_RGB9E5_MANTISSA          (RGB9E5_MANTISSA_VALUES-1)
+//#define MAX_RGB9E5                   ((f32(MAX_RGB9E5_MANTISSA))/RGB9E5_MANTISSA_VALUES * (1<<MAX_RGB9E5_EXP))
+//#define EPSILON_RGB9E5_              ((1.0/RGB9E5_MANTISSA_VALUES) / (1<<RGB9E5_EXP_BIAS))
```

Comment:
Thank you for adding how the constants were calculated. I really appreciate this!

Reply:
<3